### PR TITLE
Added 6 dpo icons in svg format

### PR DESCRIPTION
**Pull request by linwor**
** Checklist **
- [x ] All icons have a corresponding entry in `db/payment_icons.yml`
- [x ] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [ x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x ] I am confident that all icons are clear and easy to read/understand
- [x ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by appinlet** [comment]
@linwor, looking forward to seeing this live! :)

File: app/assets/images/payment_icons/dpo-xpay.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-xpay.svg:1]: We are missing the border around the payment icon.

[Video demonstration](https://screenshot.click/22-31-ogwj0-i8uh3.mp4)

File: app/assets/images/payment_icons/dpo-vodaphonempesa.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" role="img"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-vodaphonempesa.svg:1]: The border radius doesn't match other payment icons.

[Video demonstration](https://screenshot.click/22-32-hluvs-aisg8.mp4)

File: app/assets/images/payment_icons/dpo-tigopesa.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-tigopesa.svg:1]: The border radius doesn't match other payment icons.

File: app/assets/images/payment_icons/dpo-orangemoney.svg [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-orangemoney.svg:1]: We are missing the border around the payment icon.

File: app/assets/images/payment_icons/dpo-mtnmobilemoney.svg [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-mtnmobilemoney.svg:1]: The border radius doesn't match other payment icons.

File: app/assets/images/payment_icons/dpo-airtelmoney.svg [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/dpo-airtelmoney.svg:1]: The border radius doesn't match other payment icons.

Comment (linwor) [reply app/assets/images/payment_icons/dpo-airtelmoney.svg]: Icons have been updated

**Comment by appinlet** [comment]
This should now be good to go?

**Pull request closed without merging**